docs/src/main/resources/documentation/interfaces/dmf-api.md

```diff
@@ -155,12 +155,24 @@ Possible actionStatus
 |-----------------------------|----------------------------------|-------------------------------------|----------------
 | content_type                 | The content type of the payload  | String                              | true
 
-**Example Header**
+Payload Template
+
+```json
```

Comment:
The markdown is somehow broken now. There is missing an code tag ending? You can view it here:
https://github.com/bsinno/hawkbit/blob/dead262950f507021f6ab6ba9218ddbe6cc7cbb2/docs/src/main/resources/documentation/interfaces/dmf-api.md